### PR TITLE
adding liver metabolic zone as parent class of hepatic acinus zones (…

### LINKED ISSUE #1639: Grouping term 'metabolic zone of liver'
**Preferred term label:**
metabolic zone of liver

**Synonyms**
hepatic acinus metabolic zone

**Definition (free text, please give PubMed ID)**
Any of the three metabolic zones found in the hepatic acinus. Liver tissue is heterogenous at the level of morphometry and histochemistry. This heterogeneity is linked to the position of a cell within the functional unit of the tissue, which, in turn, is related to the blood supply: Cells located in the upstream zone differ from those in the downstream zone with respect to subcellular structures, key enzymes, translocators, and receptors and, therefore, have different metabolic capacities. This is the basis for “metabolic zonation,” a concept first proposed for carbohydrate metabolism and later expanded to include amino acid and ammonia metabolism, xenobiotic metabolism, protective metabolism, and formation of plasma proteins.

[Jungermann and Keitzmann (1996)](https://pubmed.ncbi.nlm.nih.gov/8839925/), PMID:8839925

**Parent term (use https://www.ebi.ac.uk/ols/ontologies/uberon)**
[organ subunit](https://www.ebi.ac.uk/ols/ontologies/uberon/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FUBERON_0000063)

**Your nano-attribution (ORCID)**
0000-0003-2034-601X

Also cc @cmungall for comment.

### PR DESCRIPTION
**Pull request by mkeays**
…fixes #1639)

### CODE REVIEW

**Review by dosumis — APPROVED** [review]

**Comment by dosumis** [comment]
@cmungall @matentzn - fail on character encoding here (see travis) showing the brittleness of the pipeline (can't we just be 100% UTF-8!).  Having trouble tracking it down.

File: uberon_edit.obo [hunk 1]
```diff
@@ -193951,40 +193951,84 @@ created_by: http://orcid.org/0000-0001-9897-3238
...
 [Term]
 id: UBERON:8400002
-name: hepatic acinus perivenous zone
-def: "The region of the hepatic acinus closest to the central vein of the liver lobule. Blood in this zome has the lowest oxygen concentration of the three metabolic zones in the hepatic acinus." [PMID:28126520]
-synonym: "hepatic acinus centrilobular zone" EXACT [PMID:28126520]
-synonym: "hepatic acinus pericentral zone" EXACT [PMID:28126520]
-synonym: "hepatic acinus zone 3" EXACT [PMID:28126520]
+name: hepatic acinus zone 3
+def: "The region of the hepatic acinus immediately surrounding the central veins. This region has the lowest oxygen concentration in the acinus. Hepatocytes in this region are the primary location for the biotransformation of drugs." [PMID:29763190]
+synonym: "liver lobule centrilobular region" RELATED [] {RELATED="PMID:28126520"}
```
Comment (dosumis) [inline uberon_edit.obo:193967]: wrong AP for xref here

File: uberon_edit.obo [hunk 2]
```diff
@@ -193951,40 +193951,84 @@ created_by: http://orcid.org/0000-0001-9897-3238
...
+relationship: adjacent_to UBERON:8400001 ! hepatic acinus zone 1
+relationship: adjacent_to UBERON:8400002 ! hepatic acinus zone 3
 relationship: dc-creator http://orcid.org/0000-0003-2034-601X
 relationship: part_of UBERON:0001172 ! hepatic acinus
 property_value: http://purl.org/dc/elements/1.1/date 2020-09-08T15:15:57Z xsd:dateTime
 
+[Term]
+id: UBERON:8400005
+name: metabolic zone of liver
+def: "Any of the metabolic zones found in the hepatic acinus. Liver tissue is heterogenous at the level of morphometry and histochemistry. This heterogeneity is linked to the position of a cell within the functional unit of the tissue, which, in turn, is related to the blood supply: Cells located in the upstream zone differ from those in the downstream zone with respect to subcellular structures, key enzymes, translocators, and receptors and, therefore, have different metabolic capacities. This is the basis for “metabolic zonation,” a concept first proposed for carbohydrate metabolism and later expanded to include amino acid and ammonia metabolism, xenobiotic metabolism, protective metabolism, and formation of plasma proteins." [PMID:8839925]
```
Comment (balhoff) [inline uberon_edit.obo:193990]: @dosumis the problem is probably the fancy quotes on this line.

Comment (dosumis) [inline uberon_edit.obo:193990]: Change smart quotes to regular double quotes to fix Travis job,

**Review by dosumis — CHANGES_REQUESTED** [review]

File: uberon_edit.obo (hunk 2 above)
Comment (dosumis) [reply uberon_edit.obo:193990]: You beat me to it! Just found these.

Comment (balhoff) [reply uberon_edit.obo:193990]: Oh, you found it simultaneously. :-)

File: uberon_edit.obo (hunk 1 above)
Comment (dosumis) [reply uberon_edit.obo:193967]: should be database_cross_reference

Comment (mkeays) [reply uberon_edit.obo:193967]: Thanks for spotting! Fixed in bec1dd0ece8421de59f5acb5a08b048317bbadf7

File: uberon_edit.obo (hunk 2 above)
Comment (mkeays) [reply uberon_edit.obo:193990]: fixed in bec1dd0ece8421de59f5acb5a08b048317bbadf7

**Review by dosumis — APPROVED** [review]

**Pull request merged**